tests/chmod/11.t

```diff
@@ -63,6 +63,12 @@ for type in regular fifo block char socket symlink; do
 		create_file ${type} ${n1} 0640 65534 65534
 		expect 0 symlink ${n1} ${n2}
 		case "${os}" in
+		Darwin)
```

Comment:
It would be fine with me if pjdfstest only passes on the latest version of each supported OS.  Managing expected failures should be the responsibility of downstream consumers, IMHO.